### PR TITLE
Enable the admin button in staging (for probers, when we have them).

### PR DESCRIPTION
**Pull request by ndmckinley**
### Description
This enables the admin button in staging.

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: universal-application-tool-0.0.1/app/modules/SecurityModule.java [hunk 1]
```diff
@@ -186,7 +186,7 @@ protected Config provideConfig(
     if (adClient != null) {
       clientList.add(adClient);
     }
-    if (URI.create(this.baseUrl).getHost().equals("localhost")) {
+    if (FakeAdminClient.canEnable(URI.create(baseUrl).getHost())) {
```
Comment (hmhsu) [inline universal-application-tool-0.0.1/app/modules/SecurityModule.java:189]: Thoughts (non-blocking): slightly surprised this is not enabled in our browser tests docker environment so far. What does it do? Why our browser tests don't seem to have problems with it?

**Review by hmhsu — APPROVED** [review]

**Comment by hmhsu** [comment]
The browser test failure is annoying. It feels flaky to me (we don't get the same result every time and I've seen this failure and others before).

File: universal-application-tool-0.0.1/app/modules/SecurityModule.java (hunk 1 above)
Comment (ndmckinley) [reply universal-application-tool-0.0.1/app/modules/SecurityModule.java:189]: I think it is enabled there - I think that's also `localhost`.

**Comment by ndmckinley** [comment]
Ah, thanks!  I spent a lot of time trying to find the bug, haha.  Let me know when you're good to go there.

**Comment by ndmckinley** [comment]
Yeah, these tests sometimes pass on my machine, so I'll rerun this until they succeed I suppose.

**Comment by hmhsu** [comment]
#693 should fix the browser tests.

File: universal-application-tool-0.0.1/app/modules/SecurityModule.java (hunk 1 above)
Comment (hmhsu) [reply universal-application-tool-0.0.1/app/modules/SecurityModule.java:189]: I see. `baseUrl` is from config not the request so it is still `localhost`. Make sense.

File: universal-application-tool-0.0.1/app/auth/FakeAdminClient.java [hunk 2]
```diff
@@ -13,11 +13,16 @@
...
 
-  public static final ImmutableSet<String> ACCEPTED_LOCALHOSTS =
-      ImmutableSet.of("localhost", "civiform");
+  public static final ImmutableSet<String> ACCEPTED_HOSTS =
+      ImmutableSet.of("localhost", "civiform", "staging.seattle.civiform.com");
 
   private ProfileFactory profileFactory;
 
+  public static boolean canEnable(String host) {
+    return FakeAdminClient.ACCEPTED_HOSTS.stream()
```
Comment (kendrickt) [inline universal-application-tool-0.0.1/app/auth/FakeAdminClient.java:22]: (somewhat related comment) TIL `ImmutableSet` uses `AbstractCollection.contains` and just iterates.

**Pull request merged**